app/views/coronavirus/announcements/_form.html.erb

```diff
@@ -11,25 +11,25 @@
 
 <%= render "govuk_publishing_components/components/input", {
   label: {
-    text: "Enter the path to the announcement",
+    text: t("coronavirus.announcements.form.path.label"),
     bold: true,
   },
   name: "announcement[path]",
   value: @announcement.path,
   id: "path",
   error_message: error_items(@announcement.errors.messages, :path),
-  hint: "Example format: /government/news/coronavirus-covid-19",
+  hint: t("coronavirus.announcements.form.path.hint"),
 } %>
 
 <% legend = capture do %>
-  <span class="govuk-heading-s govuk-!-margin-bottom-0"><%= "Enter the date of publication" %></span>
+  <span class="govuk-heading-s govuk-!-margin-bottom-0"><%= t("coronavirus.announcements.form.date.label") %></span>
```

Comment:
Might be better to refer to this as `date.legend` since it isn't technically used for a label.